google/cloud/forseti/services/cli.py

```diff
@@ -252,7 +252,7 @@ def define_model_parser(parent):
         help='Deletes an entire model')
     delete_model_parser.add_argument(
         'model',
-        help='Model to delete')
+        help='Model to delete, either hash or name')
```

Comment:
Instead of `hash`, use `handle`, as it is more consistent with usage everywhere else.

Reply:
Updated.